### ISSUE TITLE
Cannot install

### ISSUE THREAD

**Issue by FranklinChen** [andyarvanitis/purescript-native#33]
```
$ stack --version
Version 2.2.0, Git revision aa3a5f2573ffe8791f63e621447c9084d5aa0ba1 (dirty) PRE-RELEASE x86_64 hpack-0.31.1
$ stack install
Cannot complete repo information for a non SHA1 commit due to non-reproducibility: Git repo at https://github.com/purescript/purescript, commit v0.12.3
```

**Comment by andyarvanitis** [comment]
Thanks, I'll see about switching it from the tag to a hash

**Comment by andyarvanitis** [comment]
Fixed in commit d386114f764f3e23f1f93ddb4261045721179a0d "Use commit hash instead of tag for purs dependency"
